src/tools/sfdk/sdkmanager.cpp

```diff
@@ -1353,21 +1353,18 @@ bool SdkManager::mapEnginePaths(QString *program, QStringList *arguments, QStrin
         QString hostPath;
         QString enginePath;
         Qt::CaseSensitivity cs;
-    } const mappings[] = {
-#if Q_CC_GNU <= 504 // Let's check if it is still needed with GCC > 5.4
-        {cleanSharedTarget + QLatin1String("/"), QLatin1String("/"), Qt::CaseSensitive},
-        {cleanSharedTarget, QLatin1String("/"), Qt::CaseSensitive},
-        {cleanSharedHome, QLatin1String(Constants::BUILD_ENGINE_SHARED_HOME_MOUNT_POINT), Qt::CaseSensitive},
-        {cleanSharedSrc, QLatin1String(Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT), caseInsensitiveOnWindows}
-#else
-        {cleanSharedTarget + "/", "/", Qt::CaseSensitive},
-        {cleanSharedTarget, "/", Qt::CaseSensitive},
-        {cleanSharedHome, Constants::BUILD_ENGINE_SHARED_HOME_MOUNT_POINT, Qt::CaseSensitive},
-        {cleanSharedSrc, Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT, caseInsensitiveOnWindows}
-#endif
     };
+    std::vector<Mapping> mappings;
+    if (!cleanSharedTarget.isEmpty()) {
+        mappings.push_back({cleanSharedTarget + "/", "/", Qt::CaseSensitive});
+        mappings.push_back({cleanSharedTarget, "/", Qt::CaseSensitive});
+    }
+    mappings.push_back({cleanSharedHome, Constants::BUILD_ENGINE_SHARED_HOME_MOUNT_POINT, Qt::CaseSensitive});
+    mappings.push_back({cleanSharedSrc, Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT, caseInsensitiveOnWindows});
 
     for (const Mapping &mapping : mappings) {
+        if (mapping.hostPath.isEmpty())
+            continue;
```

Comment:
Not needed anymore, but maybe worth to keep in form of QTC_ASSERT.

Reply:
Replaced with QTC_ASSERT